### PR TITLE
Negative cases for TomEE

### PR DESCRIPTION
**Pull request by jannahsherif**

### CODE REVIEW

**Comment by m-reza-rahman** [comment]
Please resolve conflicts.

**Comment by jannahsherif** [comment]
I just resolved them. I did SE 8 just to make sure i am going in the right direction

File: .github/workflows/nightly.yml
Comment (m-reza-rahman) [inline .github/workflows/nightly.yml:1]: Looking at the TomEE validation: https://github.com/eclipse-ee4j/starter/blob/039a1ae96a28d43024fc347d97982ee4cda741f8/archetype/src/main/resources/META-INF/archetype-post-generate.groovy#L74, I see that the lack of EE 10 support negative test case (https://github.com/eclipse-ee4j/starter/blob/039a1ae96a28d43024fc347d97982ee4cda741f8/archetype/src/main/resources/META-INF/archetype-post-generate.groovy#L75) is missing. Please add that.

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -292,6 +299,46 @@ jobs:
...
+          fi
+          
+          rm -f mvn_output.txt   
+
+      - name: Run Archetype for EE 9.1, SE 8, TomEE
+        run: |
+          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=9.1 -DjavaVersion=8 -Druntime=tomee -DoutputDirectory=app/tomee -Dgoals="clean package" | tee mvn_output.txt
+          
+          MAVEN_EXIT_CODE=${PIPESTATUS[0]}
+          ERROR_MESSAGE="TomEE requires Web Profile and Jakarta EE 8 or 9.1."
```
Comment (m-reza-rahman) [inline .github/workflows/nightly.yml:320]: The error messages you need to check for will be from the Archetype, not the UI. Please look here: https://github.com/eclipse-ee4j/starter/blob/master/archetype/src/main/resources/META-INF/archetype-post-generate.groovy.

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -292,6 +299,46 @@ jobs:
...
+          fi
+          
+          rm -f mvn_output.txt
+          
+      - name: Run Archetype for EE 9.1 Web Profile, SE 8, TomEE
+        run: |
+          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=9.1 -Dprofile=web -DjavaVersion=8 -Druntime=tomee -DoutputDirectory=app/tomee -Dgoals="clean package" | tee mvn_output.txt
+          
+          MAVEN_EXIT_CODE=${PIPESTATUS[0]}
+          ERROR_MESSAGE="TomEE requires Web Profile and Jakarta EE 8 or 9.1."
```
Comment (m-reza-rahman) [inline .github/workflows/nightly.yml:333]: The error messages you need to check for will be from the Archetype, not the UI. Please look here: https://github.com/eclipse-ee4j/starter/blob/master/archetype/src/main/resources/META-INF/archetype-post-generate.groovy.

File: .github/workflows/nightly.yml [hunk 3]
```diff
@@ -254,6 +254,13 @@ jobs:
           docker rmi test-image
           rm -rf app/payara
 
+      - name: Run Archetype for EE 8, SE 8, TomEE
+        run: |
+          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -DjavaVersion=8 -Druntime=tomee -DoutputDirectory=app/tomee -Dgoals="clean package" | tee mvn_output.txt
+          
+          MAVEN_EXIT_CODE=${PIPESTATUS[0]}
+          ERROR_MESSAGE="TomEE requires Web Profile and Jakarta EE 8 or 9.1."
```
Comment (m-reza-rahman) [inline .github/workflows/nightly.yml:262]: The error messages you need to check for will be from the Archetype, not the UI. Please look here: https://github.com/eclipse-ee4j/starter/blob/master/archetype/src/main/resources/META-INF/archetype-post-generate.groovy.

**Review by m-reza-rahman — CHANGES_REQUESTED** [review]
The error messages and use cases are not correct. They need to correspond to the Archetype validation. I suggest getting guidance from Tasnim and/or Elzeni.

**Pull request closed without merging**